src/Domain/RequestInquiry.php

```diff
@@ -84,22 +84,22 @@ private function __construct(array $requestInquiry)
 
         $this->created = new \DateTimeImmutable($requestInquiry['created'], $timezone);
         $this->updated = new \DateTimeImmutable($requestInquiry['updated'], $timezone);
-        $this->time_responded = is_null($requestInquiry['time_responded']) ? null : new \DateTimeImmutable($requestInquiry['time_responded'], $timezone);
-        $this->time_expiry = is_null($requestInquiry['time_expiry']) ? null : new \DateTimeImmutable($requestInquiry['time_expiry'], $timezone);
+        $this->timeResponded = is_null($requestInquiry['timeResponded']) ? null : new \DateTimeImmutable($requestInquiry['timeResponded'], $timezone);
```

Comment:
I think the API still returns snake cased format, while my library uses camelCased versions.
https://doc.bunq.com/api/1/call/request-inquiry/method/get

So if you could change the API responses back, it looks good to me :+1: